src/Shouldly.Tests/ShouldNotContain/StringContainsStringCaseIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldNotContain
+{
+    public class StringContainsStringCaseIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Shouldly is legendary".ShouldNotContain("legendary", Case.Insensitive);
```

Comment:
`"Shouldly is legendary".ShouldNotContain("LEGENDARY", Case.Insensitive)` throws an exception passing the WobblyTest (providing the wobbly error message is also updated to an upper-case "LEGENDARY" as the message thrown is `"Shouldly is legendary" should not contain "LEGENDARY" but does`.

I wonder whether it's worth updating the error message that's thrown to highlight that it's case insensitive to prevent possible confusion, or should that be left up to the user with the context that was recently added?